flow/core/vehicles.py

```diff
@@ -10,6 +10,8 @@
 import numpy as np
 
 import traci.constants as tc
+VAR_TIME = 102
```

Comment:
What do these magic variables mean? Can we import them from tc instead?

Reply:
+1 see `traci.constants`

Reply:
Solved.